packages/graphql-server/src/resolvers/applicants/api.ts

```diff
@@ -0,0 +1,106 @@
+import qs from 'query-string'
```

Comment:
I think you're missing some services,
I see a total of 8 API endpoints in swagger but this API file only has 4
<img width="979" alt="Screen Shot 2020-08-10 at 18 32 18" src="https://user-images.githubusercontent.com/26746668/89778590-fb0f7580-db37-11ea-9439-9ca987d09715.png">